src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy

```diff
@@ -792,6 +796,38 @@ class PartialUpdateSpec extends HealthCheckSpecification {
         defaultFlow && flowHelperV2.deleteFlow(defaultFlow.flowId)
     }
 
+    @Tidy
+    @Unroll("Unable to partial update flow (maxLatency #maxLatencyAfter and maxLatencyTier2 #maxLatencyT2After)")
+    def "Unable to partial update flow with maxLatency incorrect value"() {
+        given: "Two potential flows"
+        def flow = flowHelperV2.randomFlow(topologyHelper.switchPairs[0]).tap {
+            maxLatency = maxLatencyBefore
+            maxLatencyT2Before
```

Comment:
`maxLatencyTier2 = maxLatencyT2Before` ?

Reply:
right
its not crucial for the case, but it was missed
fixed now